File: src/main/java/kr/co/vilez/share/model/dto/ShareDto.java
package kr.co.vilez.share.model.dto;

import io.swagger.annotations.ApiModel;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.util.List;

@Getter
@Setter
@ToString
@NoArgsConstructor
@ApiModel(value = "ShareDto : 공유 게시글", description = "공유 게시글의 상세정보를 나타낸다.")
public class ShareDto {
    int id;
    int userId;
    String category;
    String title;
    String content;
    String date;
    String hopeAreaLat;
    String hopeAreaLng;
    String startDay;
    String endDay;
    List<ImgPath> list;
    int state;

}

File: app/src/main/java/kr/co/vilez/data/model/ShareDto.kt
package kr.co.vilez.data.model

data class ShareDto(
    val area: String,
    val category: String,
    val content: String,
    val date: String,
    val endDay: String,
    val hopeAreaLat: String,
    val hopeAreaLng: String,
    val id: Int,
    val list: List<ImgPath>,
    val manner: Int,
    val nickName: String,
    val path: Any,
    val startDay: String,
    val state: Int,
    val title: String,
    val userId: Int,
    val address: String,
)